src/index.html

```diff
@@ -5,9 +5,42 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
```

Comment:
Check the rules of [HTML FORMATTING](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting) and correct all code according to these rules
```suggestion
    <link 
      rel="preconnect" 
      href="https://fonts.gstatic.com"
      crossorigin="#"
    >
```